compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/ComposeWindow.uikit.kt

```diff
@@ -152,7 +152,12 @@ internal actual class ComposeWindow : UIViewController {
 
     override fun loadView() {
         val skiaLayer = createSkiaLayer()
-        val skikoUIView = SkikoUIView(skiaLayer).load()
+        val skikoUIView = SkikoUIView(
+            skiaLayer = skiaLayer,
+            pointInside = { point, _ ->
+                !layer.hitInteropView(point, isTouchEvent = true)
```

Comment:
How oftern does uikit calls `_ pointInside` ? Does it do it on every touch event?

Reply:
Yes, it calls every native iOS touch event.

Reply:
Compose on UIKit rendered in SkikoUIView. And this view intercepts all touch events.
To disable interception of some events, - we need such poinInside function, that call before `pressesBegan(presses, event)`

So, we can't intercept event - after this event has already passed to `pressesBegan`

Reply:
But, I will research how we can reuse **hitTest** function